app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -57,25 +56,31 @@ class PocketVideoFragment : Fragment() {
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        fun displayFeed(layout: View, videos: List<PocketVideo>) {
-            layout.videoFeed.gridView.adapter = PocketVideoAdapter(context!!, videos, fragmentManager!!)
+        val layout = inflater.inflate(R.layout.fragment_pocket_video, container, false)
+
+        fun initAdapter(): PocketVideoAdapter {
+            val adapter = PocketVideoAdapter(context!!, fragmentManager!!)
+            layout.videoFeed.gridView.adapter = adapter
+            return adapter
         }
 
-        val layout = inflater.inflate(R.layout.fragment_pocket_video, container, false)
+        val videoObserver = initAdapter().VideoObserver()
 
         job = launch(UI) {
             if (deferredVideos.isNotCompleted()) {
-                //TODO set up loading UI
+                val placeholders = (1..20).map { i ->
```

Comment:
nit: 20 because we get 20 items from the server? Add a comment (or constant that is commented)